3DAmsterdam/Assets/Netherlands3D/ScriptableObjects/ConfigurationFile.cs

```diff
@@ -16,6 +17,9 @@ public enum TmsTileNumberingType
         }
 
         [Header("Coordinates")]
+
+        public Vector2RD RelativeCenterRD;
+
         public float zeroGroundLevelY = 43.0f;
```

Comment:
this is a bit confusing now:
  relativeCenterRD are the RDcoordinates at unity-Origin
  zeroGroundLevelY is the Unity-Y-axis value of the NAP-plane

wouldn't it be better if we have a Vector3RD for RdAtUnityORigin?

Reply:
or a float for NAP-elevation of the UnityOrigin?